app/helpers/mustache_helper.rb

```diff
@@ -281,6 +342,69 @@ def content
     }
   end
 
+  def settings
+    {
+      language: {
+        form: {
+          action: root_url + "/settings/language",
+          method: 'PUT'
+        },
+        title: "Language Settings",
+        language_default: {
+          title: "Default Language",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.